### ISSUE TITLE
Inaccessible icon buttons

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3118]
Several items in the "Scene" menu are accompanied by one or more actionable icons,  such as: 
- A "compass" `<svg>` to bring the item into focus in the space viewer; 
- A "wrench" icon (`<i> with icon="build"`) which opens an "Asset Information" overlay in the space viewer area; and  
- An "info" icon (`<i> with icon="help_outline"`) with a question mark in a circle, which  opens an overlay in the viewer area with the item's properties. 

The controls are not keyboard focusable or operable and are incorrectly nested inside the parent menu item `<header> with role="button"` as detailed in the previous  observation. Also the accessible name of the  control is either missing or provided using a non-descriptive text alternative for the icon. 

**Suggested Solution** 
Use native` <button>` elements for all icon buttons and apply an aria-label attribute on  each button to provide an informative accessible name. 

Severity: 1

**Comment by engbergandreas** [comment]
Closed with https://github.com/OpenSpace/OpenSpace/commit/e3f5b83f7476c8507dccd45ab98e4731504af532
